tests/test_incorrect_base_class.py

```diff
@@ -0,0 +1,34 @@
+"""Tests for incorrect base class checker."""
+import astroid
+from pylint.testutils import CheckerTestCase
+from pylint.testutils import MessageTest
+
+from pylint_nautobot.incorrect_base_class import NautobotIncorrectBaseClassChecker
+
+
+class TestIncorrectBaseClassChecker(CheckerTestCase):
+    """Test model label construction checker."""
+
+    CHECKER_CLASS = NautobotIncorrectBaseClassChecker
+
+    def test_incorrect_base_class(self):
+        test_code = "from django.db.models import Model\n\n\nclass MyModel(Model):\n    pass"
```

Comment:
I am not sure I am 100% satisfied with this representation - can we use something where the formatting is preserved somehow? I think if we get bigger statements this might get a little convoluted.

Reply:
In this case, keeping the test code in separate files makes better sense, will fix it.

Reply:
Fixed